omero/sysadmins/unix/server-installation.rst

```diff
@@ -449,13 +467,13 @@ Configuration
 
 -   You can view a parsed version of the configuration properties
     under :doc:`/sysadmins/config` or parse it
-    yourself with :program:`omero config parse`.
+    yourself with ``omero config parse``.
```

Comment:
We have a ton of these in the docs, could be fixed in another PR.

Reply:
I fixed this one since I was reviewing that page